assemble/conf/templates/generic_logger.xml

```diff
@@ -25,7 +25,7 @@
      <param name="MaxBackupIndex" value="10"/>
      <param name="Threshold"      value="DEBUG"/>
      <layout class="org.apache.log4j.PatternLayout">
-       <param name="ConversionPattern" value="%d{ISO8601} [%-8c{2}] %-5p: %m%n"/>
+       <param name="ConversionPattern" value="%d{ISO8601} [%-8c{2}] [pid:%6X{pid}] %-5p: %m%n"/>
```

Comment:
Another option would be to remove the log rolling from the log4j configuration and let the user decide whether or not to use something like logrotate